src/test/java/com/aws/greengrass/secretmanager/crypto/CrypterTest.java

```diff
@@ -5,21 +5,49 @@
 
 package com.aws.greengrass.secretmanager.crypto;
 
+import com.aws.greengrass.dependency.State;
+import com.aws.greengrass.deployment.DeviceConfiguration;
+import com.aws.greengrass.lifecyclemanager.Kernel;
 import com.aws.greengrass.secretmanager.exception.SecretCryptoException;
+import com.aws.greengrass.security.SecurityService;
+import com.aws.greengrass.security.provider.pkcs11.PKCS11CryptoKeyService;
+import com.aws.greengrass.security.provider.pkcs11.softhsm.HSMToken;
+import com.aws.greengrass.security.provider.pkcs11.softhsm.SoftHSM;
```

Comment:
Why is softHSM coming from this package? Seems odd.

Reply:
It is a test utility provided by the PKCS11 plugin. I don't want to be duplicating logic everywhere we want to test with PKCS11